src/library/entry.cr

```diff
@@ -117,6 +117,7 @@ class Entry
           sizes << {
             "width"  => size.width,
             "height" => size.height,
+            "margin" => Config.current.page_margin,
```

Comment:
This method is supposed to return the width and height of all pages in an entry, and I don't think it makes sense to have the margin value here. Do you think you can remove this line and add it to the dimensions API instead? The API route is shown below:

https://github.com/hkalexling/Mango/blob/f7b8e2d852bd90b2130437c62f8f001451f557fc/src/routes/api.cr#L648-L669

Something like 

```cr
send_json env, {
  "success"    => true,
  "dimensions" => sizes,
  "margin" => Config.current.page_margin,
}.to_json
```

would do. You can then access the value from JS using `data.margin`.

Reply:
Sure! I added the changes in this commit: https://github.com/hkalexling/Mango/pull/136/commits/aa707522449746aa88bc8f27b6e3107f2f976a2e